ui/src/app/cohort-search/redux/actions/service.ts

```diff
@@ -119,6 +123,35 @@ export class CohortSearchActions {
     this.addId(newId);
     return newId;
   }
+  updateTemporal(flag: boolean, groupId: string, role: keyof SearchRequest) {
+    this._updatedTemporal(flag, groupId);
+    const group = getGroup(groupId)(this.state);
+    if (this.hasActiveItems(group)) {
```

Comment:
This needs to check both temporalGroups for active items

Reply:
done